std/format.d

```diff
@@ -7084,14 +7084,14 @@ if (is(T == float) || is(T == double) || (is(T == real) && T.mant_dig == double.
     final switch (f.spec)
     {
         case 'a': case 'A':
-            return printFloatA(buf, val, f, rm, sgn, exp, mnt, is_upper);
+            return printFloatA(buf, val, f, rm, sgn, exp, mnt, is_upper ? 1 : 0);
         case 'e': case 'E':
             return printFloatE(buf, val, f, rm, sgn, exp, mnt, is_upper);
     }
 }
 
 private auto printFloatA(T, Char)(return char[] buf, T val, FormatSpec!Char f, RoundingMode rm,
-                                  string sgn, int exp, ulong mnt, bool is_upper)
+                                  string sgn, int exp, ulong mnt, int is_upper)
```

Comment:
It doesn't really matter that much here, but I've previously advocated for not pessimizing code legibility just because it helps the DMD optimizer; this bool -> int conversion being such a pessimization.

Reply:
It was not meant for helping DMD. I did this, because I dislike multiplying bools - feels somewhat C-ish. But I can revert this part, if you think it's better. I've got little experience here.